packages/front-end/services/metrics.tsx

```diff
@@ -336,6 +336,13 @@ export function getExperimentMetricFormatter(
 
   // Fact metric
   switch (metric.metricType) {
+    case "dailyParticipation":
+      return (() => {
+        if (metric.displayAsPercentage) {
+          return formatPercent;
+        }
+        return formatNumber;
+      })();
```

Comment:
question: why is this an `iife` and not just `if/return`?

Reply:
Typescript was complaining the first way I did it, I must have had a typo or something. I have no idea why. I'll simplify, thanks.